frontend/src/pages/_login/welcome/index.js

```diff
@@ -180,7 +181,7 @@ export default () => {
                     <img
                         className={classes.contentLeftLogo}
                         src={require('assets/logos/wordmark_white.png')}
```

Comment:
I think we need to figure out a way to properly make the logos customisable. Maybe the best option would be to have people upload them to Cloudinary, which is being used anyway, and then the customisation would just be e.g. config.LOGO_URL

Reply:
I mean it doesn't really matter if they upload it to Cloudinary or somewhere else but you get the point

Reply:
Yep, right now I just overwrote the files on the fork's master branch, but that will make it hard to do other PRs in the future. Probably referencing all of them later by URL will be the way to go.